auto_nag/scripts/component.py

```diff
@@ -71,24 +71,27 @@ def get_bz_params(self, date):
 
     def get_bugs(self, date="today", bug_ids=[]):
         # Retrieve bugs to analyze.
-        bugs, probs = super().get_bugs(date=date, bug_ids=bug_ids)
+        bugs = super().get_bugs("component", date=date, bug_ids=bug_ids)
         if len(bugs) == 0:
             return {}
 
-        # Get the encoded component.
-        indexes = probs.argmax(axis=-1)
-        # Apply inverse transformation to get the component name from the encoded value.
-        suggestions = self.model.clf._le.inverse_transform(indexes)
-
         results = {}
-        for bug, prob, index, suggestion in zip(bugs, probs, indexes, suggestions):
+
+        for bug_id in sorted(bugs.keys()):
+            bug_data = bugs[bug_id]
+            bug = bug_data["bug"]
+            prob = bug_data["prob"]
+            index = bug_data["index"]
+            suggestion = bug_data["suggestion"]
```

Comment:
I didn't realize we called it "suggestion", maybe it would be nicer to call it "class".

Reply:
Sure, will make the change

Reply:
PR: https://github.com/mozilla/bugbug/pull/670

Reply:
We need to remember to update this after the next release.